🚀 [DS-784] Update EventCard component

cbxalves opened the pull request:
[![JIRA Issue](https://img.shields.io/badge/issue-JIRA-blue.svg)](https://gympass.atlassian.net/browse/DS-784)

<!--
Please consider using title EMOJIS in the PR title to favor visualisation
PR Title Pattern: ${CLASSIFICATION EMOJI} ${PULL REQUEST TITLE}

Classification emojis
  💣 Breaking Change - Critical
  🚀 Just another PR
  🐞 Hot Fixes
-->

## Description 📄

<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->

Adjust the `EventCard` component to be aligned with the rebranding design by doing the following updates:
- Change ButtonLink color 
- Remove the shadow
- Add border

## Platforms 📲

<!-- Mark an `x` in the boxes that apply. You can also fill these out after
creating the PR.-->

- [x] Web
- [x] Mobile

## Type of change 🔍

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? 🧪

<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. -->



- [ ] Unit Test
- [x] Snapshot Test

## Checklist: 🔍

- [ ] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [x] Layout matches design prototype: [FIGMA](https://www.figma.com/file/RathBRnBqGIpeBTIMisJkj/%5BRebranding%5D-Profile?type=design&node-id=19%3A4415&mode=design&t=tcJqu73IM6mdtoeB-1)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Screenshots 📸

<!--
Load here screenshots for this PR
-->
**Event Card**
| Before (Native)  | After (Native)  |
| ------ | ----- |
|    <img width="290" alt="image" src="https://github.com/gympass/yoga/assets/60697831/87a0a0ff-0754-4ece-bb62-cc469439c5ab"> |      <img width="290" alt="image" src="https://github.com/gympass/yoga/assets/60697831/3ba238f5-d926-4760-a4fc-a809e6b5e24f"> |


| Before (Web)  | After (Web)  |
| ------ | ----- |
|    ![image](https://github.com/gympass/yoga/assets/60697831/2f59f45e-d95d-4dcf-9448-0cb9397bcb83) |      ![image](https://github.com/gympass/yoga/assets/60697831/3fab4641-3800-45b2-832c-97d6e78a265d) |

**Simple Card**

| Before (Native)  | After (Native)  |
| ------ | ----- |
|  ![image](https://github.com/gympass/yoga/assets/60697831/993ef074-b5bc-4b89-a466-5469959faf9b) |     ![image](https://github.com/gympass/yoga/assets/60697831/e45bd197-ca34-4768-936d-f94efa509b25) |
|  ![image](https://github.com/gympass/yoga/assets/60697831/f27eda16-ed03-4349-bf93-86433a9ddefc) | ![image](https://github.com/gympass/yoga/assets/60697831/9133edbb-e63a-4e89-877e-8e1ff1f7b072) |


| Before (Web)  | After (Web)  |
| ------ | ----- |
|  ![image](https://github.com/gympass/yoga/assets/60697831/bd8dedf5-a97a-46cb-8233-8d1998c032c8) |  ![image](https://github.com/gympass/yoga/assets/60697831/895e1c74-dd7f-466e-a8ef-b3b92dd19abe)|


**Plan Card**

| Before (Native)  | After (Native)  |
| ------ | ----- |
|   ![image](https://github.com/gympass/yoga/assets/60697831/a5eb0a25-7d28-46ee-af93-802a5d178d21) |     ![image](https://github.com/gympass/yoga/assets/60697831/2c5fa100-e19e-4d68-a023-7b5d3115b56c) |

| Before (Web)  | After (Web)  |
| ------ | ----- |
|  ![image](https://github.com/gympass/yoga/assets/60697831/629f54dc-d039-4c72-b885-75b7afec9ec2) |  ![image](https://github.com/gympass/yoga/assets/60697831/1b300c33-6f9f-42c5-a62d-b43ae36d1b29)  |


**Check-in Card**

| Before (Native)  | After (Native)  |
| ------ | ----- |
|  ![image](https://github.com/gympass/yoga/assets/60697831/b7aea2e7-4a9e-433a-b186-4f728eebb384)|     ![image](https://github.com/gympass/yoga/assets/60697831/b3f2befa-d321-4e83-b3f3-f7bd80dd0b3e) |

cbxalves commented on packages/yoga/src/Card/native/EventCard/EventCard.jsx:
```diff
@@ -9,15 +9,17 @@ import Text from '../../../Text';
   padding: ${spacing.zero};
   width: 280px;
   margin-right: ${spacing.xxsmall};
+  border: ${borders.small}px solid ${colors.elements.lineAndBorders};
```
Currently, the `colors.elements.lineAndBorders` color does not match the value displayed in Figma (#D7D7E0 vs #EFECE7), but I decided to use the token listed there. Is the `lineAndBorders` color going to be updated to `#EFECE7` ?

<img width="674" alt="image" src="https://github.com/gympass/yoga/assets/60697831/3a40639b-d164-47e0-bea4-cec9806d1fcb">

rafaelcoletagympass commented:
@cbxalves as we changed the component packages/yoga/src/Card/native/Card/Card.jsx (that may impact the other cards) an you please add screenshots of other cards to double check if they were impacted?

cbxalves commented:
> @cbxalves as we changed the component packages/yoga/src/Card/native/Card/Card.jsx (that may impact the other cards) can you please add screenshots of other cards to double check if they were impacted?

Sure @rafaelcoletagympass. I've added screenshots of the other cards, for both Native and Web versions. Can you take a look?
